test.sh.in

```diff
@@ -346,6 +346,23 @@ fetchOVALog(){
     popd
 }
 
+###########################################
+#
+#  Ensure logs are dumpped even build timeout aborted
+#
+#############################################
+general_cleanup(){
+
+    set +e
+    generateSysLog
+    generateMongoLog
+    vnc_record_stop
+    cleanupVMs
```

Comment:
Thank you , good catch, @PengTian0 

I've fixed it in https://github.com/RackHD/on-build-config/pull/195/commits/9e920c271321df79820958ded51ddd2f4df105f4

can you please review again ?